Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -35,7 +35,13 @@ private struct BrowserViewControllerUX {
     private static let BookmarkStarAnimationOffset: CGFloat = 80
 }
 
+protocol BrowserViewControllerDelegate: class {
+    func browserViewControllerAddReleaseNotesOnUpdate(browserViewController: BrowserViewController, profile: Profile)
```

Comment:
The delegate shouldn't be aware of what clients are doing with the implementation, so you should use a generic event name (e.g., `browserViewControllerDidAppear`).